kplclient.go

```diff
@@ -62,6 +76,37 @@ func processChannel() {
 	}
 }
 
+func (c *KPLClient) processErrMessage() {
+	time.Sleep(time.Second * 5)
+	conn, err := net.Dial("tcp", fmt.Sprintf("%s:%s", c.ErrHost, c.ErrPort))
+	if err != nil {
+		fmt.Println("Error listening to error port:", err.Error())
+		return
+	}
+
+	// Close the listener when the application closes.
+	defer conn.Close()
+
+	log.Println("Error Socket Connection Established")
+
+	for {
+		//Read from err to socket
+		content, err := Read(conn)
+		if err != nil {
+			log.Printf("Listener: Read error: %v", err)
+			continue
```

Comment:
What we should probably do is wrap this entire function in a `for` loop in order to handle reconnects, and change this line to `break`.